src/Components/Header/constants/config.js

```diff
@@ -0,0 +1,34 @@
+import search from "../../../assets/icons/search.svg";
+import offers from "../../../assets/icons/offer.svg";
+import help from "../../../assets/icons/help.svg";
+import signIn from "../../../assets/icons/sign-in.svg";
+import cart from "../../../assets/icons/cart.svg";
+
+export const navConfig = [
```

Comment:
this file could be splitted into two files each having data separately and it could be default exported

Reply:
Ya but any certain gain in doing so?